fed-budget-chart/css/lines.css

```diff
@@ -0,0 +1,210 @@
+  no-repeat 95% 50%;
+  background-position: right;
+  appearance: none;
+  background-color: transparent;
+  cursor: pointer;
+  width: fit-content;
+  min-width: 375px;
+  height: calc(100% - 0.4rem - 1px);
+  padding: 0.2rem 0.6rem;
+  font-family: "Source Sans Pro";
+  font-size: 14px;
+  background-color: #fff;
+  border: 2px solid #e7e7e7;
+  border-radius: 0;
+  /*box-shadow: inset 0 1px 3px 0 rgba(0, 0, 0, 0.05);*/
+}
+
+@media screen and (min-width: 48em) {
+  footer {
+    grid-row: 3;
+    grid-column: 2;
+    align-self: center;
+  }
+
+  #hcContainer {
+    grid-row: 2 / 4;
+  }
+}
+
+.highcharts-axis-labels.highcharts-yaxis-labels {
+  font-weight: bold;
+}
+
+.highcharts-axis-labels.highcharts-xaxis-labels {
+  font-weight: bold;
+}
+
+#legend {
+  position: relative;
+  display: block;
+}
+
+@media screen and (min-width: 48em) {
+  #legend {
+    position: relative;
+    display: block;
+    max-width: 400px;
+    padding: 16px;
+    height: 341px;
+    font-family: "Source Sans Pro", sans-serif;
+    font-size: 14px;
+    /*border: solid red 1px;*/
+  }
+}
+
+.checkboxes {
+  max-height: 300px;
+  overflow: scroll;
+  overflow-x: hidden;
+  margin-bottom: 1rem;
+}
+
+input[type='checkbox'] {
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
+  clip: rect(1px, 1px, 1px, 1px);
+}
+
+.checkbox__wrapper label {
+  color: #000;
+}
+
+.checkbox__wrapper label::before {
+  content: '';
+  position: relative;
+  display: inline-block;
+  margin-left: 10px;
+  margin-right: 10px;
+  width: .5rem;
+  height: .5rem;
+  outline: 2px solid var(--color);
+  background: rgb(255,255,255);
+}
+
+input[type='checkbox']:checked + label::before {
+  background-color: var(--color);
+  box-shadow: inset 0 0 0 0.15em #fff;  
+  outline: 2px solid var(--color);
+  width: .5rem;
+  height: .5rem;
+}
+
+input[type='checkbox']:focus + label::before {
+  outline: #000 solid 2px;
+}
+
+
+#buttons {
+  text-align: center;
+}
+
+button {
+  margin: 1 .5rem 1 .5rem;
+  padding: .3rem .5rem;
+  text-transform: uppercase;
+  font-family: 'Source Sans Pro';
+  border: 0;
+  transition: color 0.3s ease-in-out, background 0.3s ease-in-out;
+}
+
+#submit {

```

Comment:
Can you add a 2px border that is the same color as the background to the submit button? Right now it's a little bit smaller than the other two because it doesn't have a border.